src/cli/README_DATASET.md

```diff
@@ -346,7 +346,7 @@ Done
 
 ### active
 
-Usage: `dataset active [-x]`
+Usage: `dataset active [-x|nonsensitive]`
```

Comment:
We already have `-x` here (which outputs the dataset as a byte array).

Should we consider adding something short like `-s` for this (instead of `-nonsensitive`)?

Reply:
Changed to a shorter parameter -ns.